experiments/ViterBrain/scripts/napari_gui.py

```diff
@@ -1,45 +1,58 @@
 from pathlib import Path
+from matplotlib.pyplot import colorbar
 from networkx.algorithms.operators.unary import reverse
 import numpy as np
 import napari
 import pickle
 import networkx as nx
+from scipy.fft import next_fast_len
+from sklearn.manifold import trustworthiness
 from tqdm import tqdm
 import os
 from napari_animation import AnimationWidget
 from brainlit.utils.Neuron_trace import NeuronTrace
 import zarr
+from magicgui import widgets
 
 scale = [0.3, 0.3, 1]
 num = 0
 
 root_dir = Path(os.path.abspath(""))
 data_dir = os.path.join(root_dir, "data", "example")
-path = os.path.join(data_dir, "image_viterbrain.pickle")
+path = os.path.join(data_dir, "3-1-soma_viterbrain.pickle")
 
 with open(path, "rb") as handle:
     viterbi = pickle.load(handle)
 
-im_path = os.path.join(data_dir, "image.zarr")
+im_path = os.path.join(data_dir, "3-1-soma.zarr")
 z = zarr.open(im_path, "r")
 im = z[:, :, :]
 print(f"Image shape: {im.shape}")
-fragment_path = os.path.join(data_dir, "image_labels.zarr")
+fragment_path = os.path.join(data_dir, "3-1-soma_labels.zarr")
 z = zarr.open(fragment_path, "r")
 new_labels = z[:, :, :]
+print(f"Labels shape: {new_labels.shape}")
 
 state2centroid = {}
 for soma_frag in viterbi.soma_fragment2coords.keys():
     state2centroid[soma_frag] = np.mean(viterbi.soma_fragment2coords[soma_frag], axis=0)
 
 viewer = napari.Viewer(ndisplay=3)
+viewer_copy = viewer
```

Comment:
Is this variable necessary? Why have `viewer_copy` and `viewer` which point to the same thing?